src/Workspaces/Remote/ServiceHub/Services/LanguageServer/LanguageServer.cs

```diff
@@ -127,6 +127,8 @@ private static async Task<ImmutableArray<SymbolInformation>> SearchDocumentAsync
         /// </summary>
         private static async Task SearchDocumentAndReportSymbolsAsync(Document document, WorkspaceSymbolParams args, CancellationToken cancellationToken)
         {
+            Contract.ThrowIfNull(args.PartialResultToken);
```

Comment:
Are we supposed to be handling this in some way?

Reply:
This just came up on another PR, I've logged https://github.com/dotnet/roslyn/issues/48151 to fix in future. For now this is just responding to the nullable annotations that came in with the new version of the package, and making the previous behaviour a contract